manubot/cite/tests/test_handlers.py

```diff
@@ -0,0 +1,11 @@
+from ..handlers import _generate_prefix_to_handler, prefix_to_handler
+
+
+def test_prefix_to_handler():
+    """
+    If this test fails, copy the output from `print(expected)`
```

Comment:
Maybe this answers my prior question about where to document how to maintain this.